dockerfile/agoric/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ghcr.io/strangelove-ventures/infra-toolkit:v0.1.0 AS infra-toolkit
+RUN addgroup --gid 1025 -S heighliner && adduser --uid 1025 -S heighliner -G heighliner
+
+FROM ghcr.io/agoric/agoric-3-proposals:main
```

Comment:
@toliaqat - could this be turned into an `ARG AGORIC_BASE_IMAGE` such that someone could run something akin to:
```
heighliner build -c agoric -g main --build-env 'AGORIC_BASE_IMAGE="ghcr.io/agoric/agoric-3-proposals:myprivatetag"
```